HPXMLtoOpenStudio/resources/hvac_sizing.rb

```diff
@@ -2678,8 +2678,9 @@ def self.gshp_hxbore_ft_per_ton(weather, hvac_cooling_ap, bore_spacing, pipe_r_v
     rtf_DesignMon_Heat = [0.25, (71.0 - weather.data.MonthlyAvgDrybulbs[0]) / @htd].max
     rtf_DesignMon_Cool = [0.25, (weather.data.MonthlyAvgDrybulbs[6] - 76.0) / @ctd].max
```

Comment:
Are the 0 and 6 here specific to northern hemisphere? Should we use the same logic here (i.e., switch the indexes based on latitude) as we will with the ground temperatures?

Reply:
I'll take that approach for now. Let me know if it's incorrect.

Reply:
What you're doing here is correct

Reply:
Nice catch here, @joseph-robertson.